src/v/cluster/node/local_monitor.cc

```diff
@@ -62,7 +62,7 @@ ss::future<> local_monitor::update_state() {
     co_return co_await update_disk_metrics();
 }
 
-const local_state& local_monitor::get_state_cached() const { return _state; }
```

Comment:
no that's not okay. but there is some responsibility for the caller to decide if it should make a copy or interact with the reference. this change eliminates the option to avoid a copy. it's a minor point for sure as long as state remains small.

Reply:
Good to know. Still getting comfortable with this C++/seastar environment, so I'm being paranoid, and really appreciate the reviews. local_state is 64 bytes currently. I'll remove this commit.